astropy/coordinates/angles.py

```diff
@@ -426,19 +427,26 @@ def is_within_bounds(self, lower=None, upper=None):
             ok &= np.all(self < Angle(upper))
         return bool(ok)
 
-    def __str__(self):
+    def _str_helper(self, format=None):
         if self.isscalar:
-            return str(self.to_string())
```

Comment:
I removed the `str(...)` in my version, which is definitely OK for everything we support in astropy 3.0, but may need checking for backporting.